README.md

```diff
@@ -44,6 +44,17 @@ the [templates section](Longhelp.md#templates)
 You can access the HAProxy statistics via `:9090/haproxy?stats`, and you can
 retrieve the current HAProxy config from the `:9090/_haproxy_getconfig` endpoint.
 
+Marathon LB supports load balancing for applications that use the Mesos IP-per-task
```

Comment:
Could you move this section to the bottom above 'Contributing'? And give it a title. Maybe 'IP-per-task support' or something.

You can also add a bullet to the feature list at the top, like "Supports IP-per-task integration, such as Project Calico".